apps/platform/src/components/NavBar.jsx

```diff
@@ -90,58 +129,65 @@ const NavBar = ({
       color="primary"
       elevation={0}
     >
-      <Toolbar variant="dense">
-        {homepage ? null : (
-          <Button component={ReactRouterLink} to="/" color="inherit">
-            <OpenTargetsTitle name={name} />
-          </Button>
-        )}
-        <div className={classes.flex} />
-        {search ? search : null}
+      {/* push the content down so it isn't hidden behind the logo bar */}
+      {config.showTopBar &&
+          <div id="placeholder-div" style={{ height: '50px', width: '100%' }} />}
+      <Toolbar variant="dense" className={classNames(classes.spaceBetween)}>
```

Comment:
Do these lines (apart from the one above that inserts a div) have to change? I don't think they had to before, and it looks like a lot more of a hassle to maintain when EBI releases new versions, especially if they shouldn't change for the platform version.

Reply:
As we discussed in the office last night, these extra classes and wrappers were likely mostly due to differences in layout between the newer ot-ui-apps code on which the top bar was previously built and the v0.2.6 state used by OTG release 22.10. There might be cleaner solution to be found if we were to spend more time on it, but if the next release of OTG will build on a newer state of ot-ui-apps anyway (the widgets in EBI's deployment already look more modern, even though it's still on data release 22.10), my maintainability comments are irrelevant as long as we can find back the developments that we do need to keep in later releases. :+1:

Reply:
Agreed. Shall I capture this in a Trello ticket (backlog for now) to remember what we should do when the time comes?

Reply:
Neh don't think so. We will see upon a new release.